File: LostAnimalsAPI/LostAnimalsAPI/Models/Shelter.cs
using System.ComponentModel.DataAnnotations.Schema;

namespace LostAnimalsAPI.Models
{
    public class Shelter: ModelBase
    {
        public string Name { get; set; }
        public string Description { get; set; }                   
        public Location Location { get; set; }

        [NotMapped]
        public byte[] ImageResource { get; set; }
    }
}

File: LostAnimalsAndroid/app/src/main/java/com/pbl/animals/models/Shelter.java
package com.pbl.animals.models;

import android.graphics.Bitmap;
import android.graphics.BitmapFactory;

import com.pbl.animals.models.base.ModelBase;

public class Shelter extends ModelBase {
    public String name;
    public String description;
    public Location location;
    public byte[] imageSource;

    public Bitmap getImage() {
        if (imageSource == null) {
            return null;
        }

        return BitmapFactory.decodeByteArray(imageSource, 0, imageSource.length);
    }
}
